dev/galoy/main.tf

```
@@ -256,12 +256,31 @@ resource "helm_release" "postgresql" {
   ]
 }
 
+resource "random_password" "kratos_callback_api_key" {
+  length = 32
```

need to avoid special characters here:
```suggestion
  length = 32
  special = false
```